sentry/src/main/java/io/sentry/SentryClient.java

```diff
@@ -409,6 +409,13 @@ public void captureSession(final @NotNull Session session, final @Nullable Objec
         }
       }
     }
+    final List<Span> unfinishedSpans = new ArrayList<>();
+    for (Span span : transaction.getSpans()) {
+      if (!span.isFinished()) {
+        unfinishedSpans.add(span);
+      }
+    }
+    transaction.getSpans().removeAll(unfinishedSpans);
```

Comment:
Would be nice to log out. Maybe check if logging is enabled to warn level, if yes we log count of dropped span and the operations?

Reply:
I'd log it too